src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/ScheduledAppointmentRepositoryTest.java

```diff
@@ -84,6 +84,22 @@ public void returnScheduledAppointmentsByAgencyAndDate() {
                         .locationId(-28L)
                         .agencyId("LEI")
                         .createUserId("SA")
+                        .build(),
```

Comment:
A different endpoint and SQL was `seeing` the new test data - it doesn't have the same checks to avoid them.